homeassistant/components/google_assistant/trait.py

```diff
@@ -2070,3 +2073,57 @@ def query_attributes(self):
             "activityState": self.activity_lookup.get(self.state.state, "INACTIVE"),
             "playbackState": self.playback_lookup.get(self.state.state, "STOPPED"),
         }
+
+
+@register_trait
+class ChannelTrait(_Trait):
+    """Trait to get media playback state.
+
+    https://developers.google.com/actions/smarthome/traits/channel
+    """
+
+    name = TRAIT_CHANNEL
+    commands = [COMMAND_SELECT_CHANNEL]
+
+    @staticmethod
+    def supported(domain, features, device_class, _):
+        """Test if state is supported."""
+        if domain == media_player.DOMAIN and (
+            features & media_player.SUPPORT_PLAY_MEDIA
```

Comment:
Meh, this is still not ideal. Support play media doesn't mean it's supporting channels. Please limit this and also check the `device_class == media_player.DEVICE_CLASS_TV`